README.md

```diff
@@ -1,67 +1,103 @@
 # Women Coding Community Platform Backend Service
 
-## Locally Development
+<!-- TOC -->
 
-* Build and run tests
+* [Women Coding Community Platform Backend Service](#women-coding-community-platform-backend-service)
+    * [Setup](#setup)
+        * [JAVA 21 with SDKMAN](#java-21-with-sdkman)
+        * [Setup IntelliJ](#setup-intellij)
+    * [Run Locally](#run-locally)
+    * [Open API Documentation](#open-api-documentation)
 
-```shell
-./gradlew clean build
-```
+<!-- TOC -->
 
-* Create Jar
-
-```shell
-./gradlew clean bootJar
-```
+## Setup
 
-* Start Spring Boot Application:
+### JAVA 21 with SDKMAN
 
-```shell
-./gradlew bootRun
-```
-
-* Access application on http://localhost:8080/api/cms/v1/team
-
-
-
-## JAVA 21
-This project uses Java 21, you can run in 21.0.2 or 21.0.3. If you have installed a different version on your machine and don't want to remove it, you can use **SDKMAN** development tool.
-It will allow you to switch based on the Java version you want to use. 
-Here is the [link](https://sdkman.io). 
+This project uses Java 21, you can run in 21.0.2 or 21.0.3. If you have installed a different
+version on your machine and don't want to remove it, you can use **SDKMAN** development tool.
+It will allow you to switch based on the Java version you want to use.
+Here is the [link](https://sdkman.io).
 
 * Install SDKMAN
 
 Open your terminal and run the following command:
+
 ```shell
 curl -s "https://get.sdkman.io" | bash
 source "$HOME/.sdkman/bin/sdkman-init.sh"
 ```
 
 * Check the list of available Java versions:
+
 ```shell
 sdk list java
 ```
 
 * Install the desired Java version
+
 ```shell
 sdk install java 21.0.2-open 
 ```
+
 * Use the specific java version in the current session on your terminal
+
 ```shell
 sdk use java 21.0.2-open
 ```
 
-## Set the default Java version for your system:
+Set the default Java version for your system:
 
 * To set the newly installed Java version as the default:
+
 ```shell
 sdk default java 21.0.2-open
 ```
+
```

Comment:
Either remove this statement how to verify your java version or add the java - version command 
Which i can see has been removed below.

Reply:
```suggestion

* Check the java 

```shell
java -version
```
```